tests/aws/services/ssm/test_ssm.snapshot.json

```diff
@@ -1,6 +1,20 @@
 {
   "tests/aws/services/ssm/test_ssm.py::TestSSM::test_get_parameter_by_arn": {
-    "recorded-date": "16-07-2024, 17:17:40",
+    "recorded-date": "14-11-2024, 14:21:20",
+    "recorded-content": {
+      "Parameter": {
+        "ARN": "arn:<partition>:ssm:<region>:111111111111:parameter/<name:1>",
+        "DataType": "text",
+        "LastModifiedDate": "<datetime>",
+        "Name": "<name:1>",
+        "Type": "String",
+        "Value": "test",
+        "Version": 1
+      }
+    }
+  },
+  "tests/aws/services/ssm/test_ssm.py::TestSSM::test_get_parameter_by_name": {
+    "recorded-date": "14-11-2024, 14:21:20",
```

Comment:
I referred to the following documentation to update the snapshot😀 If there are any mistakes, please let me know.
https://github.com/localstack/localstack/blob/master/docs/testing/integration-tests/README.md
https://github.com/localstack/localstack/blob/master/docs/testing/parity-testing/README.md